### DISCUSSION TITLE
PerceptualHash.SumSquaredDistance

### DISCUSSION THREAD

**Discussion by HEIC-to-JPEG-Dev** [dlemstra/Magick.NET#1412] [Help] [answered]
I have the pHash of two files, and I'm trying to determine the percent difference between the two. I has hoping to use SumSquaredDistance, but I'm unsure of what the actual result means ?

I get numbers ranging from 25 to 203 ?

I can't find any documentation on this, sorry.

**Comment by dlemstra** [comment]
I also don't have that information. Maybe you could ask your question in the ImageMagick repository?

**Comment by HEIC-to-JPEG-Dev** [comment, marked as answer]
I'm not sure there is a "SumSquaredDistance" method in ImageMagick - couldn't find it in their source code, issues, or discussions.

Maybe a better question is "how do I find the percent difference between two images based on cached hashes" ?
